src/Symfony/Component/Serializer/Tests/Fixtures/FalseBuiltInDummy.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Symfony\Component\Serializer\Tests\Fixtures;
```

Comment:
the license header is missing on this file

Reply:
Added :+1: